app/routes/task_routes.py

```diff
@@ -0,0 +1,132 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.task import Task
+# from sqlalchemy import asc, desc, select
+import datetime
+# from datetime import date
+import requests
+import os
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response(
+            {"Message": f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+    if not model:
+        abort(make_response(
+            {"Message": f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
```

Comment:
Because `goal_routes.py` and 'task_routes.py` both use a function like this, we can move this function to a file and import it.